src/library/scala/util/control/ControlThrowable.scala

```diff
@@ -33,3 +33,33 @@ package util.control
  *  @author Miles Sabin
  */
 trait ControlThrowable extends Throwable with NoStackTrace
+
+object ControlThrowable {
+  sealed trait CacheableBase extends ControlThrowable {
+    override def fillInStackTrace(): Throwable = this
+  }
+
+  class Cacheable(message: String)
```

Comment:
This ought to be `CacheableThrowable`, right?

Reply:
Yes, but (a) `ControlThrowable` already has `Throwable` in its name, and (b) it's the "default", so I wanted it to have a shorter name.